components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -4296,6 +4296,36 @@ def searchProcessing():
             search.close()
         return rv
 
+    def getThumbnailSet(self, image_ids, max_size=64):
```

Comment:
as far as I understood is views responsibility to request right size https://github.com/openmicroscopy/openmicroscopy/pull/4687#issuecomment-253154356, so is everyone ok to change to 96 in a gateway now?